eva_submission/eload_ingestion.py

```diff
@@ -226,7 +226,7 @@ def load_from_ena_from_project_or_analysis(self, analysis_accession=None):
         command = (f"perl {cfg['executable']['load_from_ena']} -p {self.project_accession} -c submitted -v 1 "
                    f"-l {self._get_dir('scratch')} -e {str(self.eload_num)}")
         if analysis_accession:
-            command += f' -A -a {analysis_accession}'
+            command += f' -A 1 -a {analysis_accession}'
```

Comment:
This is an error that was compensated by another in [metadata](https://github.com/EBIvariation/metadata/pull/44/files)